### PR TITLE
runtime: format code with biome

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

**Comment by hhugo** [comment]
I'd prefer to not reformat the ts file (zstd.ts) because it's not maintained here.

**Comment by smorimoto** [comment]
@hhugo Updated!

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "formatter": {
+    "enabled": true,
+    "useEditorconfig": true
+  },
+  "linter": {
+    "enabled": false
+  },
+  "organizeImports": {
+    "enabled": true
```
Comment (hhugo) [inline biome.json:15]: What is this about ?

**Comment by TyOverby** [comment]
Could we avoid making big formatting changes until `wasm_of_ocaml` is merged?

**Comment by vouillon** [comment]
> Could we avoid making big formatting changes until `wasm_of_ocaml` is merged?

We are just going to copy the JavaScript runtime code without making any change, so this is not a problem.

**Comment by hhugo** [comment]
> > Could we avoid making big formatting changes until `wasm_of_ocaml` is merged?
> 
> We are just going to copy the JavaScript runtime code without making any change, so this is not a problem.

@vouillon, why do you need such copy for wasm_of_ocaml?

File: biome.json (hunk 1 above)
Comment (smorimoto) [reply biome.json:15]: It wasn't really necessary, but the functionality to reorder imports.

**Comment by vouillon** [comment]
What I mean is that we are going to directly use the version of the runtime files which are currently in the Js_of_ocaml repository rather than trying to merge them with those in the Wasm_of_ocaml repository.

**Pull request merged**